### PR TITLE
DDF-3633 updated docs for metacard attribute editing

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Updates documentation in these areas:
- Configuring: showing hiding metacards with data validation errors from users.
- Data management: editing and overriding metacards
- Developing: edits to creating an attribute injection file.

#### Who is reviewing it? 

@brendan-hofmann @ahoffer @mcalcote @vinamartin 

#### Select relevant component teams: 
https://github.com/orgs/codice/teams/data/members

#### Choose 2 committers to review/merge the PR. 
@beyelerb
@brendan-hofmann
@clockard
@coyotesqrl
@lessarderic
@ricklarsen - Documentation
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)

review content additions and edits.

#### What are the relevant tickets?
[DDF-3633](https://codice.atlassian.net/browse/DDF-3633)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_running/editing-attributes.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+:type: dataManagement
+:status: published
+:parent: Validating Data
+:order: 01
+:summary: Manually editing attributes.
+
+== {title}
+
+For small numbers of metacards, or for metacards ingested without overrides, attributes can be edited directly.
+For small numbers of metacards, or for metacards ingested without overrides, attributes can be edited directly.
```
Comment (clockard) [inline distribution/docs/src/main/resources/content/_running/editing-attributes.adoc]: Remove duplicate line.

Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_running/editing-attributes.adoc]: fixed.

**Comment by ricklarsen** [comment]
build now

File: distribution/docs/src/main/resources/content/_devComponents/attribute-injection.adoc [hunk 2]
```diff
@@ -40,23 +42,68 @@ In that case it can be thought of as a 'global' attribute injection.
...
+  ]
 }
 ----
 
 [NOTE]
 ====
 Attributes must be registered in the attribute registry (see the `AttributeRegistry` interface) in order to be injected into metacard types. For example, attributes defined in JSON definition files are placed in the registry, so they can be injected.
 ====
+
+Add a second key for `attributeTypes` to register the new types defined previously. For each attribute injections, specific the name and properties for that attribute.
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/content/_devComponents/attribute-injection.adoc]: ✏️ For each attribute injections, **specify** the

File: distribution/docs/src/main/resources/content/_running/correcting-errors.adoc [hunk 3]
```diff
@@ -0,0 +1,10 @@
+:title: Validating Data
+:type: dataManagementIntro
+:status: published
+:summary: Methods of correcting data validation errors.
+:order: 01
+
+== {title}
+
+${branding} enables administrators to isolate metacards with data validation errors and warnings to edit the metacard to correct validation errors.
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/content/_running/correcting-errors.adoc]: ✏️ ${branding} enables administrators to isolate metacards with data validation **issues and** edit the metacard to correct validation errors.

**Comment by ricklarsen** [comment]
yarn errors. rerunning
build now

**Pull request merged**